### ISSUE TITLE
time for pretraining step

### ISSUE THREAD

**Issue by umangkeshri** [iwangjian/textsum-gan#8]
It is mentioned in the repo that the pretraining step should run for some time, please mention after how much time i should interrupt it.

Also i can't use the pretrained npz file as i'm planning to train with my custom dataset.

**Comment by iwangjian** [comment]
Actually it should be pretrain for several epochs (e.g., 2 ~ 10), then you can interrupt it.

**Comment by umangkeshri** [comment]
for [initial pretraining](https://github.com/iwangjian/textsum-gan/blob/7aaaed5b750c8571e67bbf3ae28918bc909379de/main.py#L155) step the part of code executed is [this](https://github.com/iwangjian/textsum-gan/blob/7aaaed5b750c8571e67bbf3ae28918bc909379de/trainer.py#L15), and number of epochs are not mentioned here.

**Comment by iwangjian** [comment]
Yes, you could stop running manually when executing several epochs (the epoch count is based on the data size and the batch size). I will fix it later, thanks!
